flask/Flask_GeoAPI_setup.md

```diff
@@ -0,0 +1,649 @@
+
+```python
+@api.route('/point/distance/')
+@api.param('start_lat', 'Latitude of the start point e.g. 52.52624809700062', _in="query")
+@api.param('start_lng', 'Longitude of the start point e.g. 13.4197998046875', _in="query")
+@api.param('end_lat', 'Latitude of the end point e.g. 52.50535544522142', _in="query")
+@api.param('end_lng', 'Longitude of the end point e.g. 13.366928100585938', _in="query")
+class PointToPointDistance(Resource):
+    """
+    Return the distance in kilometers between two points.
+    """
+
+    @api.doc(id='point_to_point_distance')
+    def get(self):
+        """
+        Return the distance in kilometers between two points.
+        """
+        if request.args.get('start_lat', '') and request.args.get('start_lng', '') and \
+                request.args.get('end_lat', '') and request.args.get('end_lng', ''):
+            try:
+                return geopy.distance.distance(
+                    GeopyPoint(
+                        longitude=request.args.get('start_lng', type=float),
+                        latitude=request.args.get('start_lat', type=float)
+                    ),
+                    GeopyPoint(
+                        longitude=request.args.get('end_lng', type=float),
+                        latitude=request.args.get('end_lat', type=float)
+                    )
+                 ).km
+            except Exception:
+                pass
+
+        abort(HTTPStatus.BAD_REQUEST,
+              message="Please provide a valid query e.g. with the following url arguments: "
+                      "http://127.0.0.1:4000/api/geoapi/point/distance/?start_lng=8.83546&start_lat=53.071124&end_lng=10.006168&end_lat=53.549926")
+```
+
+The `@api.param` serves RESTPlus to be able to show the query parameters in the Swagger UI. The actual arguments are accessed via `request.args.get()` and then converted to GeoPy points with which the distance is calculated in km.
+
+(Re)-Start your Server now. You'll notice how the Swagger UI is adapting to the new input variant and how nicely it's usable through the web app.
+
+### Length of a linestring
+
+The last API endpoint will consume a GeoJSON LineString object and return the length of it. This is straight-forward and similar to the first endpoint, just expecting a LineString object instead of a Polygon:
+
+```python
+@api.route('/linestring/length/')
+class LinestringLength(Resource):
+    """
+    Return if the length in meter of a given GeoJSON LineString
+    """
+
+    @api.expect(linestring_feature, validate=True)
+    @api.doc(id='linestring_length')
+    def post(self):
+        """
+        Return if the length in meter of a given GeoJSON LineString
+        """
+        try:
+            distance.VincentyDistance.ELLIPSOID = 'WGS-84'
```

Comment:
Since you used the `geodesic` distance in the first endpoint, I'd stick with the same here.